.github/workflows/whats-new.yml

```diff
@@ -23,13 +23,14 @@ jobs:
     permissions:
       contents: write
       pull-requests: write
+    if: ${{ github.repository == 'dotnet/AspNetCore.Docs' }}
```

Comment:
I generally use the organization name only, so it's easier to reuse the code in multiple repos. (Yes, I know it's a copy / paste reuse, but still...)

```suggestion
    if: ${{ github.repository_owner == 'dotnet' }}
```

Reply:
Ahh okee interesting! Thanks for the feedback :smile: